understanding/20/focus-order.html

```diff
@@ -132,9 +132,10 @@ <h2>Examples of Focus Order</h2>
             element.
          </li>
          
-         <li>A web page implements modal dialogs via scripting. When the trigger button is activated,
+         <li>A web page implements modal dialogs. When the trigger button is activated,
```

Comment:
If you change this to remove "via scripting", you should likely remove the same phrase from [the previous example on the page](https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html#:~:text=A%20web%20page%20implements%20modeless%20dialogs%20via%20scripting).